lte/gateway/python/magma/enodebd/state_machines/acs_state_utils.py

```diff
@@ -185,6 +194,7 @@ def get_object_params_to_get(
     desired_cfg: Optional[EnodebConfiguration],
     device_cfg: EnodebConfiguration,
     data_model: DataModel,
+    request_all_params: bool = False
```

Comment:
Can we add to this method documentation the usage of this parameter? Since it seems to change an important implementation for the retrieval of the eNB params, if activated.

Reply:
The argument changes the behavior of this particular function but is an already used approach within enodebd to have a function operate on all available data model parameters. (see: get_params_to_get - equivalent for this function for normal parameters, we extended the same functionality for object params too here)

Extended docstring with explanation of the newly added boolean.